install/common-pre-install/hpcm_installation-cpi.md

```diff
@@ -0,0 +1,1076 @@
+            cinstallman --set-rootfs tmpfs --node '*'
+            cm node refresh netboot -n '*'
+            ```
+
+            __NOTE:__ Kernel version can be obtained from `cinstallman --show-default-image` command.
+
+         1. Stop the auto-discovery process and power on all nodes.
+
+            ```bash
+            cm node discover disable
+            cm power status -n '*'
+            cm power on -n '*'
+            ```
+
+            __NOTE:__ Power on the nodes when 'cm power status -n '*'' lists all the nodes. if there are any errors rerun the command.
+
+            1. Check the status of nodes.
+
+            ```bash
+            cm power status -n '*'
+            ```
+
+            Example output (expected):
+
+            ```text
+            node1        : BOOTED
+            node2        : BOOTED
+            node3        : BOOTED
+            node4        : BOOTED
+            node5        : BOOTED
+            node6        : BOOTED
+            node7        : BOOTED
+            node8        : BOOTED
+            node9        : BOOTED
+            node10       : BOOTED
+            node11       : BOOTED
+            node12       : BOOTED
+            ```
+
+            __NOTE:__ We need to wait at this step till all nodes report status as "BOOTED". Now the ports to all the NCNs can be enabled on switches.
+
+         1. Create a Bond Network
+
+## Import CSM tarball
+
+### Download CSM tarball
+
+1. (`pit#`) Download the CSM tarball
+
+   - From Cray using `curl`:
+
+      > - `-C -` is used to allow partial downloads. These tarballs are large; in the event of a connection disruption, the same `curl` command can be used to continue the disrupted download.
+      > - CSM does NOT support the use of proxy servers for anything other than downloading artifacts from external endpoints. Using `http_proxy` or `https_proxy` in any way other than the following examples will cause many failures in subsequent steps.
+
+      Without proxy:
+
+      ```bash
+      curl -C - -f -o "/var/www/ephemeral/csm-${CSM_RELEASE}.tar.gz" \
+        "https://release.algol60.net/$(awk -F. '{print "csm-"$1"."$2}' <<< ${CSM_RELEASE})/csm/csm-${CSM_RELEASE}.tar.gz"
+      ```
+
+      With HTTPS proxy:
+
+      ```bash
+      https_proxy=https://example.proxy.net:443 curl -C - -f -o "/var/www/ephemeral/csm-${CSM_RELEASE}.tar.gz" \
+        "https://release.algol60.net/$(awk -F. '{print "csm-"$1"."$2}' <<< ${CSM_RELEASE})/csm/csm-${CSM_RELEASE}.tar.gz"
+      ```
+
+   - `scp` from the external server:
+
+      ```bash
+      scp "<external-server>:/<path>/csm-${CSM_RELEASE}.tar.gz" /var/www/ephemeral/
+      ```
+
+### Import tarball assets
+
+If resuming at this stage, the `CSM_RELEASE` and `PITDATA` variables are already set
+in `/etc/environment` from the [Download CSM tarball](#download-csm-tarball) step.
+
+1. (`pit#`) Extract the tarball.
+
+   ```bash
+   tar -zxvf  "${PITDATA}/csm-${CSM_RELEASE}.tar.gz" -C ${PITDATA}
+   ```
+

```

Comment:
remove line 812 to 863 


And add the below comment 

Run the csm_dep_install.sh to install all the dependency packages
cmd: sh csm_dep_install.sh  ${PITDATA}
Note: ${PITDATA} - is the path of CSM tar extracted folder